packages/caip/src/adapters/coingecko/utils.test.ts

```diff
@@ -34,6 +41,22 @@ const btc = {
   platforms: {}
 }
 
+const cosmos = {
+  id: 'cosmos',
+  symbol: 'atom',
+  name: 'Cosmos',
+  platforms: {
+    osmosis: 'IBC/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2'
+  }
+}
```

Comment:
```suggestion
const cosmos = () => ({
  id: 'cosmos',
  symbol: 'atom',
  name: 'Cosmos',
  platforms: {
    osmosis: 'IBC/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2'
  }
})
```
mock data should be provided via functions so a test can't (accidentally) mutate the test data for other tests